### PR TITLE
added px impact score for --new-bugs

### PR DESCRIPTION
**Pull request by CFields651**

### CODE REVIEW

**Comment by CFields651** [comment]
This PR add the "PX Impact Score" to the output for --new-bugs.  PX Impact Score is from the "Product Experience" tab in OCPBUGS and is a numeric "severity" that is calculated by an algorithm described here:  [[PX Prioritization Algorithm Details](https://spaces.redhat.com/display/PPE/PX+Prioritization+Algorithm+Details)](https://spaces.redhat.com/pages/viewpage.action?spaceKey=PPE&title=PX+Prioritization+Algorithm+Details )

Basically, every bugs starts out with a score of 6000.  There are 4 factors that can increase the score up to 12000: 

1. The total quantity of linked cases
2. The quantity of linked cases with a specific severity
3. The "complexity" and quantity of the  linked cases 
4. PX properties like "Impact range" and "technical impact"

**Comment by CFields651** [comment]
I'm closing this PR because if an OCPBUG has not had the custom PX properties added to it yet (it's done on a schedule, like hourly) then I get a Traceback on when trying to access those properties.  Will fix and re-open.
